Mixpanel/Mixpanel.m

```diff
@@ -191,6 +194,7 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
 
         self.decideResponseCached = NO;
         self.showSurveyOnActive = YES;
+        self.enableABTestDesigner = YES;
```

Comment:
Yes. If we can improve the gesture, it will definitely resolve both problems.
And it is enabled on Android by default. Do you know when are we going to roll out the gesture change?